joomla/joomla-cms
# [5.1] Convert mod_languages to service provider

### Summary of Changes

Converts the language switcher module to service provider.


### Testing Instructions

1. Make sure you have site module mod_languages published
2. You should see module output on frontend

![mod_languages](https://github.com/joomla/joomla-cms/assets/59166858/c0be809d-c4b2-4655-aded-ad48515aa2e7)

### Actual result BEFORE applying this Pull Request

Module works as expected

### Expected result AFTER applying this Pull Request

Module works same way as before applying pull request

### Link to documentations
Please select:
